csrc/serde/fusion_record_serde.cpp

```diff
@@ -621,7 +604,7 @@ void RecordFunctorFactory::registerAllParsers() {
         parseStateArgs(buffer->outputs()),
         data->index());
   };
-  registerParser(serde::RecordType_SizeOp, deserializeAtOpRecord);
+  registerParser(serde::RecordType_AtOp, deserializeAtOpRecord);
```

Comment:
This was a bug I fixed that I injected in a previous PR that was causing seg faults during testing of this PR.